### ISSUE TITLE
Crash after calling func fetch(in direction: SBSMMessageDirection, completionHandler: SBSMErrorHandler? = nil)

### ISSUE THREAD

**Issue by glebpreger** [smilefam/sendbird-syncmanager-ios#1]
Hi, I am trying your ios sdk and decided to use SyncManager. But I had a problem with getting channel's messages with SBSMMessageCollection. After calling

`func fetch (in direction: SBSMMessageDirection, completionHandler: SBSMErrorHandler? = nil)
`

application crashes

```
Thread 1 Queue: com.apple.main-thread (serial)
Thread 5 Queue: com.sendbird.sdk.syncmanager.message.collection.8F05A397-FA46-4589-A996-C007A45F3EA2 (concurrent)
# 0 0x0000000103d53c04 in __DISPATCH_WAIT_FOR_QUEUE__ ()
# 1 0x0000000103d535b4 in _dispatch_sync_f_slow ()
# 2 0x000000010320996c in - [SBSMMessageCollection offsetOfMessagesInDirection:] at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Message/SBSMMessageCollection.m:640
# 3 0x000000010320b11c in __61- [SBSMMessageCollection didSynchronizeChunk: limit: direction:] _ block_invoke_2 at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Message/SBSMMessageCollection by
# 4 0x000000010320a220 in __80- [SBSMMessageCollection resolvePropagatedMessagesInDirection: completionHandler:] _ block_invoke at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Message/SBSMMessage
# 5 0x0000000103d43824 in _dispatch_call_block_and_release ()
# 6 0x0000000103d44dc8 in _dispatch_client_callout ()
# 7 0x0000000103d55030 in _dispatch_lane_concurrent_drain ()
# 8 0x0000000103d4dbd8 in _dispatch_lane_invoke ()
# 9 0x0000000103d561dc in _dispatch_root_queue_drain ()
# 10 0x0000000103d56bc8 in _dispatch_worker_thread2 ()
# 11 0x000000023701117c in _pthread_wqthread ()
# 12 0x0000000237013cec in start_wqthread ()
Enqueued from com.apple.root.default-qos.overcommit (Thread 0) Queue: com.apple.root.default-qos.overcommit (serial)
# 0 0x0000000103d497e0 in dispatch_barrier_async ()
# 1 0x0000000103209f48 in - [SBSMMessageCollection resolvePropagatedMessagesInDirection: completionHandler:] at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Message/SBSMMessageCollection.m:709
# 2 0x000000010320b068 in __61- [SBSMMessageCollection didSynchronizeChunk: limit: direction:] _ block_invoke at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Message/SBSMMessageCollection.m:age
# 3 0x0000000103213f20 in - [SBSMCommand execute] at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Command/SBSMCommand.m:24
# 4 0x0000000103213d4c in - [SBSMOperation main] at /Developer/Projects/jios/SendBirdSyncManager/SyncManager/Command/SBSMOperation.m:83
# 5 0x0000000237ea923c in __NSThread__start__ ()
# 6 0x000000023701025c in _pthread_body ()
# 7 0x00000002370101bc in _pthread_start ()
# 8 0x0000000237013cf4 in thread_start ()
com.apple.uikit.eventfetch-thread (6) Thread 7com.sendbird.SocketRocket.NetworkThread (9) com.apple.NSURLConnectionLoader (10) Thread 11 Queue: com.apple.CFNetwork.adPersistCacheToStorageDaemon (serial)
Thread 12 Queue: Anonymous
com.apple.CFSocket.private (13) Thread 14

```
SendBirdSDK (3.0.130)
SendBirdSyncManager (1.0.0):
Xcode 10.1
iOS 12.1.4
Swift 4.2

**Comment by famous90** [comment]
@glebpreger Sorry for the crash. We are fixing the error you've got. We will release updated SyncManager SDK ASAP.

**Comment by famous90** [comment]
Hi @glebpreger We just released v1.0.1 of SendBirdSyncManager SDK before. Can be used without any problems. Sorry for the inconvenience.

**Comment by glebpreger** [comment]
Hi @famous90, I confirm that this problem is solved in version 1.0.1, thanks!
